specs/extras/linked-token.md

```diff
@@ -0,0 +1,206 @@
+# Linked Token Spec
```

Comment:
I think this spec is missing a high-level description. We should assume the reader has basic knowledge of IPC and then explain which use cases Linked Token addresses and explain in text the flows visualised by the diagrams.

Also, [xnet messaging spec](https://github.com/consensus-shipyard/ipc/pull/905/files#diff-f9352466600ed4d3251a7278b5fb63c46e63614d1a3592c126d518f2b12ab191R199) already goes into the details of LinkedToken. I think those details should only be here and that other spec should link to this one.